exonum-java-binding/launcher-plugin/README.md

```diff
@@ -0,0 +1,45 @@
+# Java Runtime Plugin for Exonum Launcher
+
+## Installation
+
+#### Requirements
+
+- [Python3](https://www.python.org/downloads/)
+- [`exonum-python-client`](https://github.com/exonum/exonum-python-client)
```

Comment:
Do I have to install these manually?

Reply:
For now yes, later we will provide an easier approach (after publishing on pypi)